api/v1beta2/ratelimitpolicy_types.go

```diff
@@ -160,10 +160,11 @@ func (s *RateLimitPolicyStatus) Equals(other *RateLimitPolicyStatus, logger logr
 	return true
 }
 
-// RateLimitPolicy is the Schema for the ratelimitpolicies API
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 // +kubebuilder:metadata:labels="gateway.networking.k8s.io/policy=direct"
+
+// RateLimitPolicy is the Schema for the ratelimitpolicies API
```

Comment:
```suggestion
// RateLimitPolicy enables rate limiting for service workloads in a Gateway API network (Gateways and HTTPRoutes).
```

Reply:
Thank you for the suggestions. I modified them accordingly, just had to trim the ending a bit as it was not fitting in length in the card on Operator hub. I hope that's ok.

Reply:
Example of how it fades when the description is longer:
![Screenshot from 2023-11-27 14-15-13](https://github.com/Kuadrant/kuadrant-operator/assets/97438342/6cd169b4-ef75-4f8f-9d1d-25619e10df87)